### ISSUE TITLE
Dialog: aria-describedby comes as empty if no subtext is provided

### ISSUE THREAD

**Issue by prmittal** [OfficeDev/office-ui-fabric-react#1116]
If no value is given for subText prop of Dialog component, it sends an empty value for ariaDescribedBy prop of Popup component. As a result, the aria-describedby value for the rendered component comes as empty and it fails the keros check.

![image](https://cloud.githubusercontent.com/assets/11937034/23363464/c053f1c6-fd20-11e6-9376-1d99cbd53507.png)

**Comment by mdahamiwal** [comment]
being tracked by #1088 . Closing as duplicate.
